src/cr/cube/collator.py

```diff
@@ -12,9 +12,11 @@
 
 import collections
 import sys
+from typing import Dict, FrozenSet, Iterator, Tuple, Union, List
```

Comment:
isort should order import in A-Z isn't it?

Reply:
fixed